doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -202,6 +202,12 @@ Operators typically do not implement Security Groups when ussing SR-IOV or DPDK
 <a name="6.3.5"></a>
 ### 6.3.5 Image Security
 
+OpenStack provides valuabele guidance on trusted image creation process, and image signature verification (please see "Trusted Images" section in the [OpenStack Security Guide](https://docs.openstack.org/security-guide/instance-management/security-services-for-instances.html#trusted-images/) documentation). Adding Signed Images to the Image Service (Glance) is specified in [OpenStack Operations Guide](https://docs.openstack.org/operations-guide/ops-user-facing-operations.html#adding-signed-images).
+
+Integrity Verification at instantiation is required by [ETSI NFV SEC021](https://portal.etsi.org/webapp/WorkProgram/Report_WorkItem.asp?WKI_ID=53601) and the creation of signature per individual artifact in the VNF package is required by [ETSI NFV SOL004](http://www.etsi.org/deliver/etsi_gs/NFV-SOL/001_099/004/02.03.01_60/gs_nfv-sol004v020301p.pdf).
```

Comment:
Minor typo - Verification at the time of instantiation

Reply:
@sukhdevkapur Corrected -- Thanks